portal-frontend/src/app/shared/no-data/no-data.component.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../styles/colors';
+@import '../../../styles/typography';
+
+.text {
+  color: $grey-dark;
+}
+
+.error {
+  margin-top: rem(4);
+  color: $error-color;
+  display: flex;
+  align-items: center;
+
+  .mat-icon {
+    margin-right: rem(4);
+    font-size: 14px;
```

Comment:
is this intentional in pixels and not in. rem?

Reply:
No it is not, changed to rem